posthog/temporal/batch_exports/squash_person_overrides.py

```diff
@@ -201,61 +162,27 @@ async def optimize_person_distinct_id_overrides(dry_run: bool) -> None:
 
 
 @activity.defn
-async def prepare_dictionary(inputs: DictionaryInputs) -> None:
+async def prepare_dictionary(dry_run: bool) -> None:
     """Prepare the DICTIONARY to be used in the squash workflow."""
     from django.conf import settings
 
-    create_dictionary_query = CREATE_DICTIONARY_QUERY.format(
-        database=settings.CLICKHOUSE_DATABASE,
-        dictionary_name=inputs.dictionary_name,
-        user=settings.CLICKHOUSE_USER,
-        password=settings.CLICKHOUSE_PASSWORD,
-        cluster=settings.CLICKHOUSE_CLUSTER,
-    )
-
-    if inputs.dry_run is True:
+    if dry_run is True:
         activity.logger.info("This is a DRY RUN so no dictionary will be created.")
-        activity.logger.debug("Create dictionary query: %s", create_dictionary_query)
         return
 
     async with heartbeat_every():
         async with get_client() as clickhouse_client:
-            await clickhouse_client.execute_query(create_dictionary_query)
             # ClickHouse may delay populating the dictionary until we read from it.
             # We force a reload here to ensure the values are populated. This way,
             # they remain static from this point onwards as the dictionary's lifetime
             # is 0 (no updates).
```

Comment:
The last sentence here about the dictionary remaining static doesn't seem like it would be accurate now with the background refreshes.